packages/reports/src/Containers/open-positions.jsx

```diff
@@ -360,6 +360,35 @@ const OpenPositions = ({
         contract_types.find(type => type.is_default)?.value || 'Options'
     );
 
+    const accumulator_rates = [
+        {
+            text: 'All Rates',
+            value: 'all',
+        },
+        {
+            text: '1%',
+            value: '0.01',
+        },
+        {
+            text: '2%',
+            value: '0.02',
+        },
+        {
+            text: '3%',
+            value: '0.03',
+        },
+        {
+            text: '4%',
+            value: '0.04',
+        },
+        {
+            text: '5%',
+            value: '0.05',
+        },
+    ];
+
+    const [accumulator_rate, setAccumulatorRate] = React.useState('all');
```

Comment:
```suggestion
    const [accumulator_rate, setAccumulatorRate] = React.useState(accumulator_rates[0]);
```
@henry-deriv just to avoid hardcoding the value🙂